### ISSUE TITLE
What is the password for decompressing KS20.zip ?

### ISSUE THREAD

**Issue by sieannHit** [Kali-Hac/TranSG#1]

**Comment by linhuaiyi** [comment]
@sieannHit have you got the password. Could you please tell me what the password is.

**Comment by buldings** [comment]
i want to know too

**Comment by sieannHit** [comment]
> @sieannHit have you got the password. Could you please tell me what the password is.
Please refer to the README.md for complete information about obtaining datasets, i.e., "The access to the Vislab Multi-view KS20 dataset and large-scale RGB-based gait dataset CASIA-B are available upon request. If you have signed the license agreement and been granted the right to use them, please email us with the signed agreement and we will share the complete pre-processed KS20 and CASIA-B data. ".

In other words, you need to first apply to the official for permission to obtain the datasets, and then contact the authors of "TranSG" by e-mail for requesting passwords.

**Comment by sieannHit** [comment]
> i want to know too

Please refer to the README.md for complete information about obtaining datasets, i.e., "The access to the Vislab Multi-view KS20 dataset and large-scale RGB-based gait dataset CASIA-B are available upon request. If you have signed the license agreement and been granted the right to use them, please email us with the signed agreement and we will share the complete pre-processed KS20 and CASIA-B data. ".

In other words, you need to first apply to the official for permission to obtain the datasets, and then contact the authors of "TranSG" by e-mail for requesting passwords.

**Comment by sieannHit** [comment]
> @sieannHit have you got the password. Could you please tell me what the password is.

Please refer to the README.md for complete information about obtaining datasets, i.e., "The access to the Vislab Multi-view KS20 dataset and large-scale RGB-based gait dataset CASIA-B are available upon request. If you have signed the license agreement and been granted the right to use them, please email us with the signed agreement and we will share the complete pre-processed KS20 and CASIA-B data. ".

In other words, you need to first apply to the official for permission to obtain the datasets, and then contact the authors of "TranSG" by e-mail for requesting passwords.

**Comment by ChenCherry** [comment]
I want to know too
